### DISCUSSION TITLE
[@types/bootstrap] Include "RegExp" in allow list

### DISCUSSION THREAD

**Discussion by alexwang216** [DefinitelyTyped/DefinitelyTyped#61882] [Issues with a @types package]
As the example shown in Bootstrap doc, the Tooltips.allowList accept string or RegExp.

Suggested change:
`allowList: Record<keyof HTMLElementTagNameMap, string[]>;`
To:
`allowList: Record<keyof HTMLElementTagNameMap, (string | RegExp)[]>;`

@denisname, @peterblazejewicz, @martin-badin.

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment]
PR is ready https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61885
